### DISCUSSION TITLE
Add second tunnel to same cloudflare account?

### DISCUSSION THREAD

**Discussion by ee241** [homeassistant-apps/app-cloudflared#673] [Q&A]
I have two locations with two home assistant controllers running which I would like to connect to one cloudflare account where I have both domains managed.

I get an error when connecting the second homeassistant tunnel because there is one already with a matching name, despite that these were linked to different domains:

failed to create tunnel: Create Tunnel API call failed: tunnel with name already exists

When viewing the existing tunnel, it shows locally managed and does not allow me to edit the name. Is there a way to edit the configuration of the cloudflare add--on so that the tunnel is connected under a different name?

**Comment by elcajon** [comment]
You can run multiple tunnels with the same cloud flare account, have a look here #643
